src/AWS.Deploy.Common/Exceptions.cs

```diff
@@ -110,8 +110,7 @@ public enum DeployToolErrorCode
         FailedToRunCDKDiff = 10009000,
         FailedToCreateCDKProject = 10009100,
         ResourceQuery = 10009200,
-        FailedToCreateContainerDeploymentBundleFromGeneratedDockerFile = 10009300,
-        FailedToRetrieveStackId = 10009400
+        FailedToRetrieveStackId = 10009300
```

Comment:
Why reassign that enum value? Will that break anything in Toolkit or their metrics?

Reply:
this shouldn't be an issue. right now we're only using these for telemetry, but we should be more careful in the future.

Reply:
Okay, thanks for confirming.